src/server/helpers/file-cache.ts

```diff
@@ -102,4 +118,16 @@ export default class FileCache {
     return flatCache.load(fileName(this.name_, isProd), this.path).getKey(key)
       ?.data;
   }
+  isStale() {
+    if (this.hashes.length >= 5) {
```

Comment:
Kun je de waarde 5 in een goed benoemde constant zetten bovenaan de file?

Reply:
of misschien nog beter, kun je deze waarde configurable maken in de `datasets.ts` file?